libs/components/lookup/src/assets/locales/resources_en_US.json

```diff
@@ -27,6 +27,10 @@
     "_description": "Menu item text to display the user has searched and there are more items to view",
```

Comment:
Looking at the screenshots the default one has redundant hint texts in the input box and in the dropdown, can we confirm with design that that's what we want?

Reply:
Removed placeholder text from phone field per design